components/script/dom/htmlanchorelement.rs

```diff
@@ -100,6 +102,23 @@ impl VirtualMethods for HTMLAnchorElement {
             _ => self.super_type().unwrap().parse_plain_attribute(name, value),
         }
     }
+
+    fn handle_event(&self, event: &Event) {
+        if let Some(s) = self.super_type() {
+            s.handle_event(event);
+        }
+
+        if let Some(key_event) = event.downcast::<KeyboardEvent>() {
+            if event.type_() == atom!("keydown") {
+                match key_event.get_key() {
+                    Some(Key::Enter) => {
+                        follow_hyperlink(self.upcast::<Element>(), Some(String::new()), None);
```

Comment:
We should be calling into activation_behaviour here, which will need to be updated to account for non-mouse activations. We should also handle space key events similarly.